wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/WrappersToGenerate.kt

```diff
@@ -948,6 +948,10 @@ val wrappersToGenerate = listOf(
     ),
     WrapperRequest(ActionCoords("peterjgrainger", "action-create-branch", "v2.2.0")),
 
+    WrapperRequest(
+        ActionCoords("ReactiveCircus", "android-emulator-runner", "v2.25.0")
```

Comment:
```suggestion
        ActionCoords("ReactiveCircus", "android-emulator-runner", "v2")
```
Please use major version - the action provides a `v2` tag: https://github.com/ReactiveCircus/android-emulator-runner/tree/v2